vital/types/track.cxx

```diff
@@ -79,9 +79,9 @@ ::track( const track& other )
 /// Factory function
 track_sptr
 track
-::make(track_data_sptr data)
+::create( track_data_sptr data )
```

Comment:
I don't object to this change, but I'm wondering what the motivation was.  Does this better match some naming used elsewhere?

Reply:
track_descriptor used "create" instead of "make" for the same functionality so I decided to normalize them, I liked create slightly more

Reply:
okay, that sounds reasonable.